rpcserver.go

```diff
@@ -6029,6 +6029,11 @@ func (r *rpcServer) AddAssetBuyOrder(_ context.Context,
 	req *rfqrpc.AddAssetBuyOrderRequest) (*rfqrpc.AddAssetBuyOrderResponse,
 	error) {
 
+	// TODO(ffranr): Use context as a timeout mechanism.
```

Comment:
Good idea in general. But maybe we need both mechanisms? Thinking of a REST user, I don't think they would have any way of cleanly aborting the context.

Reply:
A REST client should be able to cancel their request by closing the TCP connection, with a REST lib should support as part of request cancellation. (Closing a TCP connection sends a `FIN` message to the server.) That should translate to a cancelled context in `rpcServer.AddAssetBuyOrder`. I haven't tested that though.